docs/recipes/PWA.md

```diff
@@ -0,0 +1,339 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to Overview](./README.md)
+<!--ONE-DOCS-HIDE end-->
+
+[one-app-config]: ../api/modules/App-Configuration.md
+[frank-lloyd-root]: ../../prod-sample/sample-modules/frank-lloyd-root/0.0.3/README.md
+[one-service-worker]: https://github.com/americanexpress/one-service-worker
+
+# PWA
+
+## 📖 Table of Contents
+* [Overview](#overview)
+* [Setup](#setup)
+* [Offline Support](#offline-support)
+* [App Installation](#app-installation)
+
+## Overview
+
+In this compilation of mini recipes well be getting our feet wet with Progressive Web Apps
+(PWA) in One App. We'll learn a bit about the service worker and how to use the web
+manifest with One App. Each mini recipe below will cover UX issues and the technical
+solutions to refine your One App PWA.
+If any of these terms are new, we have links ready to get you familiar below in this section.
+We recommend going through and reviewing the links before using PWA if you haven't
+already done so.
+
+**Links**
+
+If you want to learn more about PWAs, service workers and web manifests,
+here are a collection of links that talk more about the technologies involved:
+
+- [PWA](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps)
+- [Service Worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API)
+- [Web App Manifest](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps)
+- [App Installs](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Developer_guide/Installing)
+- [Online/Offline](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/Online_and_offline_events)
+- [Permissions](https://developer.mozilla.org/en-US/docs/Web/API/Permissions_API)
+- [Cache](https://developer.mozilla.org/en-US/docs/Web/API/Cache)
+- [Cache Storage](https://developer.mozilla.org/en-US/docs/Web/API/CacheStorage)
+
+**Examples**
+
+* [`frank-lloyd-root` v0.0.3](frank-lloyd-root) sample module configures and enables PWA.
```

Comment:
Maybe `frank-lloyd-root@0.0.3` it looks weird (visually) in the readme currently with `frank-lloyd-root` and then v0.0.3 following it and that will translate to the docs site as well.

Reply:
updated (b912ca5)